src/types/BscType.ts

```diff
@@ -72,20 +72,33 @@ export abstract class BscType {
     }
 
 
-    checkCompatibilityBasedOnMembers(targetType: BscType, flags: SymbolTypeFlag) {
+    checkCompatibilityBasedOnMembers(targetType: BscType, flags: SymbolTypeFlag, data: TypeCompatibilityData = {}) {
```

Comment:
I was basically checking this backwords - making sure this type had all the members of the incoming type.